contracts/sfc/Staker.sol

```diff
@@ -369,13 +406,13 @@ contract Stakers is StakersConstants {
 
         require(delegations[delegator].amount != 0, "delegation doesn't exist");
         require(delegations[delegator].deactivatedTime == 0, "delegation is deactivated");
+        require(checkBonded(), "not passed enough time since stake is bonded");
 
         uint256 pendingRewards;
         uint256 fromEpoch;
         uint256 untilEpoch;
         (pendingRewards, fromEpoch, untilEpoch) = calcDelegationRewards(delegator, _fromEpoch, maxEpochs);
 
-        require(bondedRatio > bondedTargetRewardUnlock(), "below minimum bonded ratio");
         require(block.timestamp > capReachedDate + unbondingUnlockPeriod(), "before minimum unlock period");
```

Comment:
require(block.timestamp > capReachedDate + unbondingUnlockPeriod(), "before minimum unlock period");

block.timestamp > 0 + 6 months == true

Reply:
Should be fixed in ded1f86, please check